docs/source/intro.rst

```diff
@@ -3,14 +3,15 @@
    :maxdepth: 4 
    :caption: Contents:
 
+************
 Introduction
-------------
+************
 
 Overview
-********
+========
 
-A BLAS implementation on top of AMD’s Radeon Open Compute ROCm runtime and toolchains. 
-rocBLAS is implemented in the HIP programming language and optimized for AMD’s latest 
+A `BLAS <http://www.netlib.org/blas/>`__ implementation on top of AMD’s Radeon Open Compute `ROCm <https://rocm.github.io/install.html>`__ runtime and toolchains.
+rocBLAS is implemented in the `HIP <https://github.com/ROCm-Developer-Tools/HIP>`__ programming language and optimized for AMD’s latest 
```

Comment:
Please change to:     rocBLAS is a `BLAS <http://www.netlib.org/blas/>' implementation ....